pymel/versions.py

```diff
@@ -184,6 +184,9 @@ def bitness():
 v20023_2 = 20230200
 v20023_3 = 20230300
 v20024 = 20240000
+v20024_1 = 20240100
+v20024_2 = 20240200
+v2025 = 20250000
```

Comment:
unbelievably, these 2023 and 2024 versions seem to have an extra 0.  can you correct this, like so:

```python
v20023_1 = 20230100  # bad name.  keeping for backwards compatibility
v20023_2 = 20230200  # bad name.  keeping for backwards compatibility
v20023_3 = 20230300  # bad name.  keeping for backwards compatibility
v2023_1 = 20230100
v2023_2 = 20230200
v2023_3 = 20230300
v20024 = 20240000
v20024 = 20240000  # bad name.  keeping for backwards compatibility
v2024 = 20240000
v2024_1 = 20240100
v2024_2 = 20240200
v2025 = 20250000
```